command/command.go

```diff
@@ -57,7 +56,7 @@ func handleGetUpdatesRequest(cache *cache.Cache, guMsg *sync_pb.GetUpdatesMessag
 				}
 
 				// Error out when exceeds the limit.
-				if activeDevices >= maxActiveDevices {
+				if checkDeviceLimit(activeDevices, clientID) {
```

Comment:
To go with the suggested name below:
```suggestion
				if exceedsDeviceLimit(activeDevices, clientID) {
```